.eslintrc.js

```diff
@@ -107,11 +107,16 @@ module.exports = {
     },
     {
       files: ['**/*.test.js'],
+      excludedFiles: ['**/swaps/**/*.test.js'],
```

Comment:
Nit: we should probably specify `pages/swaps` here for consistency with the other config block.

Or even better, we could be more explicit in both places to ensure it only applies to this one directory:
```suggestion
      excludedFiles: ['ui/pages/swaps/**/*.test.js'],
```